doc/content/getting-started/cli/installing-cli/_index.md

```diff
@@ -8,11 +8,13 @@ This section contains instructions for installing the command-line interface.
 
 <!--more-->
 
+{{% tts %}} CLI is called `ttn-lw-cli`. There is also an enterprise specific CLI called `tti-lw-cli` that is nearly identical, but the enterprise `tti-lw-cli` has additional features for [tenant management]({{< ref "reference/api/tenant" >}}). All other commands are the same. For most users, including Cloud and Community Edition users, `ttn-lw-cli` is sufficient.
+
 ### Package managers (recommended)
 
-{{< tabs/container "Open Source" "Enterprise" >}}
+{{< tabs/container "All Distributions" "Enterprise" >}}
```

Comment:
Should we call this then **Cloud and Community** ?

Reply:
Are "Open Source" and "Enterprise" the "names" of these two CLI versions? Because the linked issue also mentiones "open-source and enterprise client", I'm not sure now what we should do with these tags. Would it be weird if someone using Community installed `tti-lw-cli` and found out that it works?

Reply:
Hmm. Practically speaking, the CLI is:

- `ttn-lw-cli`: Open Source, Community and Cloud
- `tti-lw-cli`: Enterprise

If we ever were to rename the CLI names, it would be `tts` and `ttse`.

Reply:
Ok, I would then indeed use "Open Source, Cloud and Community" instead of "All distributions"